jac/jaclang/compiler/passes/ecmascript/tests/test_js_generation.py

```diff
@@ -86,7 +86,8 @@ def test_core_fixture_emits_expected_constructs(self) -> None:
             "function add",
             "function greet",
             "function fibonacci",
-            "for (const i of range(limit))",
+            "for (const i of array)",
+            "for (  let i = 0; i < limit;   i += 1;)",
```

Comment:
i wonder why the spacing is weird here, not a big deal for now, but just curiuosu